nio/client/async_client.py

```diff
@@ -876,6 +876,25 @@ async def login(
 
         return await self._send(LoginResponse, method, path, data)
 
+    def restore_login(
+        self,
+        user_id: str,
+        device_id: str,
+        access_token: str,
+    ):
+        """Restores a previous login to the homeserver.
```

Comment:
Personal preference, not a big deal: I'd use "Restore" over "Restores". I think most of the other methods follow that style.